controllers/storageclient_controller.go

```diff
@@ -433,14 +433,24 @@ func (s *StorageClientReconciler) reconcileClientStatusReporterJob(instance *v1a
 	cronJob.Namespace = s.OperatorNamespace
 	addLabel(cronJob, storageClientNameLabel, instance.Name)
 	addLabel(cronJob, storageClientNamespaceLabel, instance.Namespace)
+	var jobDeadLineSeconds int64 = 155
+	var podDeadLineSeconds int64 = 120
```

Comment:
- as these two are related, pls show that in code as well like `jobSec = podSec + 35`